subscription-offer-prompt/pwr-of-you.js

```diff
@@ -3,6 +3,7 @@ import Superstore from 'superstore'
 
 import * as utils from './utils';
 import { broadcast } from 'n-ui-foundations';
+import { getCountryCode } from './countryApi'
```

Comment:
@jkerr321 moved the api call to its own file to be able to stub it out more easily in tests